src/v2i-hub/CDA1TenthPlugin/src/CDA1TenthPlugin.cpp

```diff
@@ -107,95 +103,98 @@ void CDA1TenthPlugin::HandleMobilityOperationMessage(tsm3Message &msg, routeable
 
 	// Retrieve J2735 Message
 	auto mobilityOperation = msg.get_j2735_data();
-	
+
 	// String Stream for strategy and operationParams
-	std::stringstream strat;
-	std::stringstream payload; 
- 
-	// Create ptree object for json
-	ptree pr;
+	std::string strat;
+	std::stringstream payload;
 
-	// Create new CDA1Tenth_Object pointer
-	std::unique_ptr<CDA1Tenth_Object> pd( new CDA1Tenth_Object());
+	// Get current strategy from V2X Hub config
+	GetConfigValue<string>("Mobility_Strategy", strat);
+	std::stringstream stratStream(strat);
+
+	// Create ptree object for json payload
+	ptree json_payload;
+
+	// Create new Action_Object pointer
+	std::unique_ptr<Action_Object> action_obj( new Action_Object());
 
 	// Read strategy and operationParams
-	strat << mobilityOperation->body.strategy.buf;
+	stratStream << mobilityOperation->body.strategy.buf;
```

Comment:
Is this stratStream from the manifest config or from MOM message? Shouldn't we compare the strategy between the MOM message and manifest.json rather than `PORT_DRAYAGE_STRATEGY`?